integration-tests/src/test/resources/config-test-child/configuration/domain2/file-override.png

```diff
@@ -0,0 +1 @@
+override file
```

Comment:
Not exactly best practice, but I created "png" files with text because this bug was only manifesting itself with non-filtered file types (because, I assume, filtering ends up flagging all files as "new").